lib/irb/completion.rb

```diff
@@ -388,7 +388,7 @@ def retrieve_completion_data(input, bind:, doc_namespace:)
 
         if doc_namespace
           rec_class = rec.is_a?(Module) ? rec : rec.class
-          "#{rec_class.name}#{sep}#{candidates.find{ |i| i == message }}"
+          "#{rec_class.name}#{sep}#{candidates.find{ |i| i == message }}" rescue nil
```

Comment:
:+1:
I think there is one more `class.name` in Line 421 that needs `rescue nil`

Reply:
Oops! Fixed in https://github.com/ruby/irb/pull/899/commits/00b5a9788dea0f97c2dce64ac32a2ae5c2911ade

Thanks for the review!